### PR TITLE
Support speed and course

### PR DESCRIPTION
**Pull request by kf99916**
Support speed and course according to the[ gpx spec](https://www.topografix.com/gpx_manual.asp).

### CODE REVIEW

File: gpx.go [hunk 1]
```diff
@@ -343,6 +345,16 @@ func (w *WptType) MarshalXML(e *xml.Encoder, start xml.StartElement) error {
 	if err := maybeEmitFloatElement(e, "ele", w.Ele); err != nil {
 		return err
 	}
+	if w.Speed > 0 {
+		if err := maybeEmitFloatElement(e, "speed", w.Speed); err != nil {
+			return err
+		}
+	}
+	if w.Course > 0 {
```
Comment (twpayne) [inline gpx.go]: As with `w.Speed`, this conditional can be removed.

**Comment by kf99916** [comment]
@twpayne I have updated the PR according to your suggestion. Thanks.

**Comment by twpayne** [comment]
Perfect, thank you very much!

**Comment by twpayne** [comment]
I tagged version 1.1.0 to include your contribution (in case you're using Go modules). Thanks again!

**Pull request merged**